NetSDK/Services/Client/Classes/SplitFactory.cs

```diff
@@ -89,9 +89,9 @@ private void BuildSplitClient()
                 case Mode.Consumer:
                     if (options.CacheAdapterConfig != null && options.CacheAdapterConfig.Type == AdapterType.Redis)
                     {
-                        if (String.IsNullOrEmpty(options.CacheAdapterConfig.Host) || String.IsNullOrEmpty(options.CacheAdapterConfig.Port) || String.IsNullOrEmpty(options.CacheAdapterConfig.Password))
+                        if (String.IsNullOrEmpty(options.CacheAdapterConfig.Host) || String.IsNullOrEmpty(options.CacheAdapterConfig.Port))
```

Comment:
@patricioe @dendril @sarrubia  Is it ok to remove this validation? . Let me know so I can merge this PR too